src/main/java/frc/robot/scoreStates/AmpState.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.scoreStates;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import frc.robot.Constants;
+import frc.robot.swerve.SwerveDrive;
+import java.util.Set;
+import lib.Utils;
+
+public class AmpState implements ScoreState {
+
+    @Override
+    public Command driveToClosestOptimalPoint() {
+        return Commands.defer(
+                () -> {
+                    Translation2d ampPose;
+                    if (isRed()) {
+                        ampPose = ScoreStateConstants.AMP_POSE_RED;
+                    } else {
+                        ampPose = ScoreStateConstants.AMP_POSE_BLUE;
+                    }
```

Comment:
Can be reverted to a ternary operation.

Reply:
No, the spotless reverts it back to an if statement